src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -51,12 +51,12 @@ type Props = {
     tagged_message: string,
     translatedTaggedMessage?: string,
     translations?: Translations,
-    type?: typeof COMMENT_TYPE_DEFAULT | typeof COMMENT_TYPE_TASK,
+    type: typeof COMMENT_TYPE_DEFAULT | typeof COMMENT_TYPE_TASK,
```

Comment:
Apparently when we do ` & InjectIntlProvidedProps` in a flow type (like we had in `CommentMenu`) it resolves to `any` and stops properly flow checking, which meant that this was broken before but was passing anyway. Once I removed those props this and `isConfirming?` became flow errors.

Reply:
Not sure, but it could be because it wasn't imported on top?
import type { InjectIntlProvidedProps } from 'react-intl';

I notice if you add that, it seems to show correctly, so it is a bug that we are just doing any

Reply:
Yep, I think something with the way we're doing the flow-typed lib defs in box-ui-elements.js is breaking those types in certain cases.

Reply:
Yeah, we should import the types explicitly